database/src/main/runs/checkpoints.ddl

```diff
@@ -13,21 +13,20 @@
  * limitations under the License.
  */
 
--- DROP TABLE IF EXISTS runs.checkpoints;
-
 CREATE TABLE runs.checkpoints
 (
     id_checkpoint           UUID NOT NULL,
-    key_segmentation        BIGINT NOT NULL,
+    fk_partitioning         BIGINT NOT NULL,
```

Comment:
don't forget to add author as well

Reply:
`created_by`
I know the name goes against what I wrote above. But this is widespread and rhymes with `created_at`, `updated_by` etc....